cypress/e2e/dashboard.cy.ts

```diff
@@ -32,6 +31,29 @@ describe('Dashboard', () => {
         insight.addInsightToDashboard(dashboardName, { visitAfterAdding: true })
 
         cy.get('.CardMeta h4').should('have.text', insightName)
+
+        dashboard.addPropertyFilter()
+        cy.get('main').contains("There are no matching events for this query").should('not.exist')
+
+        cy.clickNavMenu('dashboards')
+        const dashboardNameB = randomString('Dashboard with insight B')
+        dashboards.createAndGoToEmptyDashboard(dashboardNameB)
+
+        insight.visitInsight(insightName)
+        insight.addInsightToDashboard(dashboardNameB, { visitAfterAdding: true })
+
+        dashboard.addPropertyFilter("Browser", "Hogbrowser")
+
+        // Go back and forth to make sure the filters are correctly applied
+        for (let i = 0; i < 3; i++) {
+            cy.clickNavMenu('dashboards')
+            dashboards.visitDashboard(dashboardName)
+            cy.get('main').contains("There are no matching events for this query").should('not.exist')
+
+            cy.clickNavMenu('dashboards')
+            dashboards.visitDashboard(dashboardNameB)
+            cy.get('main').contains("There are no matching events for this query").should('exist')
+        }
```

Comment:
Unfortunately we need to roll back the un-skipping of this test. @webjunkie Do we need to we remove this addition too, or should it stay in for the time we _actually_ un-skip the test?

Reply:
I think I've fixed the basic `'Adding new insight to dashboard works` test, which should work even in master. We could comment out and place a TODO on the part about checking for filters being correctly applied.

Reply:
Nice, the test does pass after removing the added cases. Will merge then!